src/manager.rs

```diff
@@ -28,90 +29,89 @@ pub struct PartitionedService {
 }
 
 impl PartitionedService {
-    pub fn new(config: Arc<Config>, partition: u16) -> Self {
-        Self {
-            config,
-            partition,
-            config_store: Arc::new(ConfigStore::new_rw()),
-            shutdown_signal: CancellationToken::new(),
-        }
-    }
+    pub fn start(config: Arc<Config>, partition: u16) -> (Self, JoinHandle<()>) {
```

Comment:
Starting the partitioned service now returns the join handle for the scheduler.

We don't wait for this yet, but in a future commit we can.